crates/handler/src/handler.rs

```diff
@@ -401,9 +401,9 @@ pub trait Handler {
     fn reimburse_caller(
         &self,
         evm: &mut Self::Evm,
-        exec_result: &mut <<Self::Evm as EvmTr>::Frame as FrameTr>::FrameResult,
+        exec_result: &<<Self::Evm as EvmTr>::Frame as FrameTr>::FrameResult,
```

Comment:
This `&mut` is left there if someone wants to modify end result.

Reply:
> This `&mut` is left there if someone wants to modify end result.

Makes sense.